sonar-plugin-api/src/main/java/org/sonar/api/Property.java

```diff
@@ -109,7 +109,9 @@
    * by its key.
    *
    * @since 3.3
+   * @deprecated since 6.1, as it was not used and too complex to maintain.
    */
+  @Deprecated
   String propertySetKey() default "";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)